### DISCUSSION TITLE
Cannot find the AggregatorV3Interface in the chainlink/contracts/src/V0.8/interfaces/

### DISCUSSION THREAD

**Discussion by Owenzo7** [Cyfrin/foundry-full-course-cu#1177] [Q&A]
In the video it states that under the interfaces folder you can find the AggregatorV3Interface but I cannot find it yet

**Comment by sagoto80** [comment]
use relative path
```
import {AggregatorV3Interface} from "../../lib/chainlink-brownie-contracts/contracts/src/v0.8/interfaces/AggregatorV3Interface.sol";
```

**Reply by Owenzo7** [reply]
Hey there I'm on the remix section if you using cyfrin updraft I'm still on the solidity fundamentals I havent made it to vs code yet

**Comment by xyz899** [comment]
Hi ! 
To import AggregatorV3Interface in Remix, you typically use an import statement that points to the Chainlink GitHub repo where the interface is held. 

after that on top of your Solidity file add this below the pragma version :
`import "@chainlink/contracts/src/v0.8/interfaces/AggregatorV3Interface.sol";`

Now it should work correctly

**Reply by Owenzo7** [reply]
Thanks bro
